Modules/Core/Common/include/itkHexahedronCell.hxx

```diff
@@ -349,7 +349,7 @@ HexahedronCell<TCellInterface>::EvaluatePosition(CoordRepType *            x,
   double                  tcol[Self::PointDimension3D];
   double                  d;
   PointType               pt;
-  CoordRepType            derivs[CellDimension3D * Self::NumberOfPoints];
+  CoordRepType            derivs[CellDimension3D * Self::NumberOfPoints]{ 0 };
   InterpolationWeightType weights[Self::NumberOfPoints];
```

Comment:
Thank you.  I am not familiar enough with the code to know which variables are guaranteed to be set before being used ... should any of the other array variables also be initialized at construction?: `fcol`, `rcol`, `scol`, `tcol`, `weights`.

Reply:
I'm also not to familiar. I decided to leave the others alone as a warning was only raised for the `derivs` variable.